jsonschema-module-microprofile-openapi-3/src/test/resources/com/github/victools/jsonschema/module/microprofile/openapi3/integration-test-result-Foo.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$defs": {
+        "referenceToPerson": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description": "the person's name"
+                }
+            },
+            "title": "reference title",
+            "description": "the foo's person",
+            "additionalProperties": false
+        }
+    },
+    "type": "object",
+    "properties": {
+        "anyOfDoubleOrInt": {
+            "anyOf": [{
+                    "type": "number"
+                }, {
+                    "type": "integer"
+                }]
+        },
+        "bar": {
+            "$ref": "http://example.com/bar",
+            "readOnly": true
+        },
+        "oneOfBooleanOrString": {
+            "oneOf": [{
+                    "type": "boolean"
+                }, {
+                    "type": "string"
+                }]
+        },
+        "person": {
+            "$ref": "#/$defs/referenceToPerson",
+            "writeOnly": true
+        },
+        "test": {
+            "$ref": "./TestClass-schema.json"
+        }
+    },
+    "additionalProperties": false
```

Comment:
**suggestion**: As per my other comment, it'd be nice to avoid `"additionalProperties": false` as part of the module's default behavior.

Reply:
I am taking theses two `additionalProperties`. Needs some times to understand and fix them

Reply:
It is also the case for the [swagger 2 version](https://github.com/victools/jsonschema-generator/blob/main/jsonschema-module-swagger-2/src/test/resources/com/github/victools/jsonschema/module/swagger2/integration-test-result-Foo.json#L44) I need to check it too.
The openapi code is derived from swagger 2.

Reply:
Swagger 2 version is expected because `additionalProperties = Schema.AdditionalPropertiesValue.FALSE` is added in `Foo.class`

Mine is expected too except in `$defs` part. I am checking why

Reply:
Ok I just fixed it. A misunderstand from my side regarding additionalProperties.
I made a dedicated commit to fix it (likes others).